packages/back-end/src/util/mssqlPoolManager.ts

```diff
@@ -0,0 +1,14 @@
+import mssql from "mssql";
+import { MssqlConnectionParams } from "../../types/integrations/mssql";
+const pools = new Map();
+
+export function get(name: string, config: MssqlConnectionParams) {
+  if (!pools.has(name)) {
+    if (!config) {
+      throw new Error("Pool does not exist");
+    }
+    const pool: mssql.ConnectionPool = new mssql.ConnectionPool(config);
+    pools.set(name, pool.connect());
```

Comment:
Noticing the cleanup step from the example is not included here ... It's probably necessary. Without it, it may unearth a bug in the future if/when we do decide to close connections.

Reply:
I'm weakly in favor of adding it in even if we aren't using it. I basically am thinking of this code as something that kinda should/could be in the package we're importing and we'd probably expect this functionality.

But it just adds bloat for the foreseeable future so I'm also fine leaving it out.

Reply:
The few lines you've added @mknowlton89 look good to me